data.sql

```diff
@@ -54,4 +54,49 @@ SET owner_id = (
 
 select * from animals;
 
+  
+   INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES
+    ((SELECT id FROM animals WHERE name = 'Agumon'), 
+    (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24'),
+    ((SELECT id FROM animals WHERE name = 'Agumon'), 
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22'),
+    ((SELECT id FROM animals WHERE name = 'Gabumon'), 
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-02'),
+    ((SELECT id FROM animals WHERE name = 'Pikachu'), 
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-01-05'),
+    ((SELECT id FROM animals WHERE name = 'Pikachu'), 
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-03-08'),
+    ((SELECT id FROM animals WHERE name = 'Pikachu'), 
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-05-14'),
+    ((SELECT id FROM animals WHERE name = 'Devimon'), 
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2021-05-04'),
+    ((SELECT id FROM animals WHERE name = 'Charmander'), 
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-24'),
+    ((SELECT id FROM animals WHERE name = 'Plantmon'), 
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-12-21'),
+    ((SELECT id FROM animals WHERE name = 'Plantmon'), 
+    (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-08-10'),
+    ((SELECT id FROM animals WHERE name = 'Plantmon'), 
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2021-04-07'),
+    ((SELECT id FROM animals WHERE name = 'Squirtle'), 
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2019-09-29'),
+    ((SELECT id FROM animals WHERE name = 'Angemon'), 
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-10-03'),
+    ((SELECT id FROM animals WHERE name = 'Angemon'), 
+    (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-11-04'),
+    ((SELECT id FROM animals WHERE name = 'Boarmon'), 
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-01-24'),
+    ((SELECT id FROM animals WHERE name = 'Boarmon'), 
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-05-15'),
+    ((SELECT id FROM animals WHERE name = 'Boarmon'), 
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-02-27'),
+    ((SELECT id FROM animals WHERE name = 'Boarmon'), 
+    (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-08-03'),
+    ((SELECT id FROM animals WHERE name = 'Blossom'), 
+    (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-05-24'),
+    ((SELECT id FROM animals WHERE name = 'Blossom'), 
+    (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
+
+
 
```

Comment:
- Good job inserting the above data into the visit table, but I think you forgot to add data into the `vet` and `specialization` tables as required for this project
![Screenshot from 2023-02-23 15-05-11](https://user-images.githubusercontent.com/44624138/220931527-874d43db-b2d1-4935-ad55-99d1d1d285f5.png)

Reply:
- Well resolved 👍🏽